src/incubator/f5-bigip-ctlr/README.md

```diff
@@ -1,11 +1,25 @@
-# Alpha status
+# Helm Chart for the F5 BIG-IP Controller
 
-This chart is in an alpha status. It can be used to deploy the __development__ image of the [F5 k8s-bigip-ctlr](https://github.com/F5Networks/k8s-bigip-ctlr). The stable image will be available in a stable chart.
+This chart simplifies repeatable, versioned deployment of the [F5 BIG-IP Controller for Kubernetes](http://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/latest/).
 
-To deploy the k8s-bigip-ctlr with this chart the following must be true of your cluster:
-- A secret for the BigIP must exist and be named as in the values file, or passed using `--set` or `-f your_values.yaml`.
-- The partition must exist and be named as in the values file, or passed using `--set` or `-f your_values.yaml`.
-- The BigIP management url must be passed as an argument using `--set` or configured in a values file.
+## Requirements
+
+The chart presumes:
```

Comment:
I would change this to a Prerequisites sub-section

```
### Prerequisites
- Add your BIG-IP device to your [Kubernetes](http://clouddocs.f5.com/containers/latest/kubernetes/kctlr-use-bigip-k8s.html) or [OpenShift](http://clouddocs.f5.com/containers/v2/openshift/kctlr-use-bigip-openshift.html) Cluster.
- Create a partition on your BIG-IP device for the BIG-IP Controller to manage. The Controller cannot manage objects in the `/Common` partition.
- Create a Secret containing the BIG-IP login credentials for the Controller. The Controller needs an account with administrator-level permissions to ensure full functionality.

Reply:
Done.